jest.config.js

```diff
@@ -9,4 +9,5 @@ const config = standard.jestConfig();
 // Adding jest-environment-jsdom-fifteen because the jsdom included with @grafana/toolkit doesn't play nice with the current version of @testing-library
 config.testEnvironment = 'jest-environment-jsdom-fifteen';
 config.setupFilesAfterEnv = ['@testing-library/jest-dom/extend-expect', ...(config.setupFilesAfterEnv || [])];
+config.maxWorkers = '25%';
```

Comment:
Whaaat?! I did not know this was an option! Out CI is pretty flaky because of the default parallelism, but I was waiting for a change in `@grafana/toolkit`